docs/source/getting-started/tracking-server-overview/index.rst

```diff
@@ -327,4 +327,4 @@ of this method below:
 * **Cons**
   
   * Not free.
-  * Need to manage a billing account.
\ No newline at end of file
+  * Need to manage a billing account.
```

Comment:
not an issue, I am just confused why this is marked as a change

Reply:
My guess is a line-ending change. If it is an issue, I'm happy to revert the whole file, since there ended up being no changes in it after all was said and done.